example.js

```diff
@@ -1,19 +1,36 @@
+var fs = require('fs')
 var hyperdrive = require('hyperdrive')
 var ram = require('random-access-memory')
-var swarm = require('.')
-var Buffer = require('safe-buffer').Buffer
+var Discovery = require('.')
 
-var key = process.argv[2] && new Buffer(process.argv[2], 'hex')
+var key = process.argv[2]
 var archive = hyperdrive(ram, key)
+var archive2 = hyperdrive(ram)
+var discovery = Discovery(archive)
+
 archive.ready(function (err) {
   if (err) throw err
+  // discovery.add(archive)
```

Comment:
Why the comment?

Reply:
Had this in there before I added support for the original API implementation of `discovery(archive)`. Removed.